projects/exui-common-lib/src/lib/gov-ui/components/hmcts-main-wrapper/hmcts-main-wrapper.component.ts

```diff
@@ -13,11 +15,18 @@ import {Component, Input} from '@angular/core';
   templateUrl: './hmcts-main-wrapper.component.html'
 })
 export class HmctsMainWrapperComponent  {
+  public bannerData: BannerDataModel;
 
   @Input() public backLink: string;
   @Input() public title: string;
   @Input() public summaryErrors: {header: string; isFromValid: boolean; items: { id: string; message: any; }[]};
+  @Input() public set banner(value: BannerDataModel) {
+    this.bannerData = value;
+  }
 
-  constructor() { }
+  constructor( private readonly location: Location) { }
+  public onGoBack() {
+    this.location.back();
```

Comment:
Don't use location directly here. Component should have an EventEmitter on an @Output that relays the click on 'go back' back to the original application, which can then decide what to do (or how to handle going back).